cmd/image-builder/README.md

```diff
@@ -1,290 +1,175 @@
-you must not use rendertemplates for generating your ProwJob definition. Using `preset-image-builder-ado-token` is compatible with
-rendertemplates.
-
-Follow these steps to migrate to the ADO backend:
-
-1. Add the **ADO_PAT** environment variable to the ProwJob definition.
-   Use the predefined `image-builder-ado-token` preset in the ProwJob definition if you use rendertemplates.
-   You can remove the data file for your ProwJob to effectively stop using rendertemplates as well.
-   ```yaml
-   labels:
-     preset-image-builder-ado-token: "true"
-   ```
-   If you don't use rendertemplates, you can add the **ADO_PAT** environment variable directly.
-   ```yaml
-   env:
-     - name: "ADO_PAT"
-       valueFrom:
-         secretKeyRef:
-           name: "image-builder-ado-token"
-           key: "token"
-   ```
-2. Add the `--build-in-ado=true` flag to the Image Builder command.
-   ```yaml
-   args:
-     - "--build-in-ado=true"
-   ```
-3. Remove signify secrets from the ProwJob definition.
-4. Remove unsupported Image Builder flags from the ProwJob definition. See [Deprecated Features](#deprecated-features) for more information.
-
-### Opt Out of ADO Backend
-
-The ADO backend is going to be the only SLC-29 compliant backend. To opt out of using the ADO backend in the ProwJob, use
-the `--build-in-ado=false` flag.
+- Images build on pull requests are pushed to the dev repository, `europe-docker.pkg.dev/kyma-project/dev`.
+- Images build on push events are pushed to the production repository, `europe-docker.pkg.dev/kyma-project/prod`.
 
-```yaml
-args:
-  - "--build-in-ado=false"
+### Image URI
+
+The URI of the image built by Image Builder is constructed as follows:
+
+```
+europe-docker.pkg.dev/kyma-project/<repository>/<image-name>:<tag>
 ```
 
-## Deprecated Features
+Where:
 
-### Build Multi-Architecture Images
+* `<repository>` is the repository where the image is pushed. It can be either `dev` or `prod`, based on the event that triggered the build.
+* `<image-name>` is the name of the image provided in the `name` input.
+* `<tag>` is the tag of the image provided in the `tags` input or default tag value.
 
-> [!WARNING] 
-> This is an experimental feature that may change in the future.
+## Image Signing
 
-With the introduction of the experimental BuildKit support, the tool now supports the repeatable flag `--platform`.
-You can define multiple platforms you want to build an image for.
+Image Builder signs images with a signify service.
+By default, Image Builder signs images with the production signify service.
+Image signing allows verification that the image comes from a trusted repository and has not been altered in the meantime.
 
-You can use all platforms supported by [BuildKit](https://github.com/moby/buildkit/blob/master/docs/multi-platform.md).
+> [!NOTE]
+> Image Builder signs images build on push event only. Images build in pull_request_target event are not signed.
```

Comment:
```suggestion
> Image Builder signs images built in the `push` event only. Images built in the `pull_request_target` event are not signed.
```

Reply:
The workflow is running as a consequence of happening a `push` or a `pull_request_target` event. We don't run something in the event, rather we run something when this event occurs. Should we still use `in the` instead `on`?

Reply:
No, not really. How about `resulting from`? If you still don't like it, keep `on`. But you originally wrote `in pull_request_target event` - is this phrase different in meaning from the first one?

Reply:
I've aligned the sentence to use `on` only.